packages/@aws-cdk/aws-eks/lib/managed-nodegroup.ts

```diff
@@ -51,6 +55,32 @@ export interface NodegroupRemoteAccess {
   readonly sourceSecurityGroups?: ISecurityGroup[];
 }
 
+/**
+ * Represents the LaunchTemplate.
+ */
+export interface ILaunchTemplate {
```

Comment:
I don't think that is what @eladb meant in his comment. `IXX` in CDK are normally constructs, not just interfaces. This interfaces is not adding any benefit, just another level of nesting...

Since we don't have an L2 for `LaunchTemplate`, let's just use strings for now.

Reply:
Sure. Thanks.